sdcm/nemesis.py

```diff
@@ -1268,3 +1282,59 @@ class NodeRestartWithResharding(Nemesis):
     @log_time_elapsed_and_status
     def disrupt(self):
         self.disrupt_restart_with_resharding()
+
+
+class DisruptiveMonkey(Nemesis):
+
+    @log_time_elapsed_and_status
+    def disrupt(self):
+        # Limit the nemesis scope:
+        #  - ValidateHintedHandoffShortDowntime
+        #  - CorruptThenRepairMonkey
+        #  - CorruptThenRebuildMonkey
+        #  - RestartThenRepairNodeMonkey
+        #  - StopStartMonkey
+        #  - MultipleHardRebootNodeMonkey
+        #  - HardRebootNodeMonkey
+        #  - SoftRebootNodeMonkey
+        #  - StopWaitStartMonkey
+        #  - NodeTerminateAndReplace
+        #  - EnospcMonkey
+        #  - DecommissionMonkey
+        #  - NodeRestartWithResharding
+        #  - DrainerMonkey
+        self.call_random_disrupt_method(disrupt_methods=['disrupt_stop_wait_start_scylla_server',
```

Comment:
Maybe disruptive and non-disruptive should be a property of the nemesis and these list would be generated dynamically here.
It's easier to maintain it that way and ensuring that when people add a nemesis, they don't forget to add them to the relevant list.
We can do the same for all other general classes that holds list of nemesis instead of generating them.

Reply:
you are right, but anyway we are going to refactor Nemesis code soon, and for the matter of simplicity let's keep it as is

Reply:
implement dynamic building of method list for disruptive and nondisruptive monkeys